### DISCUSSION TITLE
Valaxy Features

### DISCUSSION THREAD

**Discussion by YunYouJun** [YunYouJun/valaxy#116] [Show and tell]
We will demonstrate Valaxy's features here by text, images or videos.

Wait and see...

---

当前的一些进展与特色预览；

```bash
// 一行命令完成安装配置启动
pnpm create valaxy
```

![image](https://user-images.githubusercontent.com/25154432/194337707-954dffce-d368-4e40-8a93-2bc7e6dfa510.png)

> 配置热更新（动画也能重载）

https://user-images.githubusercontent.com/25154432/194336670-d0b75126-6597-4d53-8861-bcbed681ffea.mov

> 内容无感知的瞬间更新

https://user-images.githubusercontent.com/25154432/194336856-f7d84aec-a3ac-4ca9-9c19-fcb65f896a92.mov

> 全局的 i18n 切换

https://user-images.githubusercontent.com/25154432/194336964-34b7beea-6a7d-4f58-9ea0-74812420fc06.mov

> 全局响应式状态

https://user-images.githubusercontent.com/25154432/194338555-0f5fb2f0-9f88-43b6-bed9-2bfaf9d2d513.mov

SSG? 这是当然的（用户也可以切换 SPA 构建）：

![image](https://user-images.githubusercontent.com/25154432/194337284-6192649a-f9fa-46db-9429-25afd92ca1f1.png)

内置的 RSS 生成 `npm run rss`：

![image](https://user-images.githubusercontent.com/25154432/194337387-3d7955af-399b-415d-834a-6ad2c6f0f9cd.png)

## 其他 Features

- 你可以细粒化地覆盖定制所使用主题的每一个组件（同时保持主题的更新，构建时也不会增加冗余的体积）
- 基于 UnoCSS 提供的所有 TailwindCSS 辅助类（按需生成），让你的定制快人一步
- 基于 Vite 的启动速度，只有一个字：快
- 任意使用 icones.js.org 中的所有图标，并按需加载其 SVG（简而言之：又小又快）
- 完全的配置类型提示
- 支持文章 Markdown 嵌入 Vue 组件、组件自动注册

**Comment by misaka-clover** [comment]
好耶😆
